modules/arrow/test/geoarrow/convert-geoarrow-to-binary-geometry.spec.ts

```diff
@@ -342,7 +342,7 @@ async function testGetBinaryGeometriesFromArrow(
 
     t.notEqual(encoding, undefined, 'encoding is not undefined');
     if (geoColumn && encoding) {
-      const options = {meanCenter: true};
+      const options = {meanCenter: true, triangulate: true};
```

Comment:
Nit: Both of these options trigger extra work, but one is a verb and one is a noun:
```suggestion
      const options = {calculateCenters: true, triangulate: true};
```